numba_dpex/core/runtime/context.py

```diff
@@ -33,10 +33,25 @@ def meminfo_alloc(self, builder, size, usm_type, device):
         return self.meminfo_alloc_unchecked(builder, size, usm_type, device)
 
     @_check_null_result
-    def meminfo_fill(self, builder, meminfo, itemsize, is_float, value, device):
+    def meminfo_fill(
+        self,
+        builder,
+        meminfo,
+        itemsize,
+        dest_is_float,
+        value_is_float,
+        value,
+        device,
+    ):
         """A wrapped caller for meminfo_fill_unchecked() with null check."""
```

Comment:
The docstring should be improved.

Reply:
We need a link to `meminfo_fill_unchecked` and properly explain what the null check is and how it works.